platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -465,19 +465,6 @@ data:
   #
   url: jdbc:h2:mem:anchor-platform
 
-  ## @param: username
-  ## @type: string
-  ## The username for database connection
-  #
-  username:
-
-
-  ## @param: username
-  ## @type: string
-  ## The password for database connection
-  #
-  password:
-
```

Comment:
Can you add comments to this section explaining which secrets will need to be configured in the env vars in order for the DB connection to work?

This file should be self-explanatory in terms of how to configure the application.

Reply:
Is that accurate? I thought you were enforcing it to use the `secret` prefix. So it'd be something like secret.data.password or SECRET_DATA_PASSWORD.

Reply:
Also, could you update example.env with the new secrets?